cmd/controller/main.go

```diff
@@ -103,6 +104,16 @@ func main() {
 		setupLogging(ctlConf.LogDir, ctlConf.LogSizeLimitMB)
 	}
 
+	logger.WithField("logLevel", ctlConf.LogLevel).Info("Setting LogLevel configuration")
```

Comment:
This all looks really good :+1:

Reply:
Thanks, we can split that up into two parts. I just scanned through the logging, when nothing happens, no events but some `syncing`, `enqueuing` appears always.
Need to add log example to the ticket.